### PR TITLE
Changing opensuse-leap16-repoindex.xml to distribution instead of repositories

### PR DESCRIPTION
**Pull request by stacheldrahtje**
Changing opensuse-leap16-repoindex.xml to distribution instead of repositories

### CODE REVIEW

File: opensuse-leap16-repoindex.xml [hunk 1]
```diff
@@ -11,19 +11,19 @@
     enabled="true"
     autorefresh="true"/>
 
-<repo url="%{disturl}/repositories/openSUSE:/Leap:/16.0/product/repo/Leap-Packages-16.0-$DIST_ARCH"
+<repo url="%{disturl}/distribution/leap/16.0/repo/oss"
```
Comment (lkocman) [inline opensuse-leap16-repoindex.xml:14]:
```suggestion
<repo url="%{disturl}/distribution/leap/%{distver}/repo/oss"
```

File: opensuse-leap16-repoindex.xml [hunk 2]
```diff
@@ -11,19 +11,19 @@
...
     alias="repo-oss"
     name="%{alias} (%{distver})"
     enabled="true"
     autorefresh="true"/>
 
-<repo url="%{disturl}/repositories/openSUSE:/Leap:/16.0/product/repo/Leap-Packages-16.0-$DIST_ARCH-Debug"
+<repo url="%{disturl}/debug/distribution/leap/16.0/repo/oss"
```
Comment (lkocman) [inline opensuse-leap16-repoindex.xml:20]:
```suggestion
<repo url="%{disturl}/debug/distribution/leap/%{distver}/repo/oss"
```

File: opensuse-leap16-repoindex.xml [hunk 3]
```diff
@@ -11,19 +11,19 @@
...
     alias="repo-oss-debug"
     name="%{alias} (%{distver})"
     enabled="false"
     autorefresh="true"/>
 
-<repo url="%{disturl}/repositories/openSUSE:/Leap:/16.0/product/repo/Leap-Packages-16.0-$DIST_ARCH-Source"
+<repo url="%{disturl}/source/distribution/leap/16.0/repo/oss"
```
Comment (lkocman) [inline opensuse-leap16-repoindex.xml:26]:
```suggestion
<repo url="%{disturl}/source/distribution/leap/%{distver}/repo/oss"
```

**Review by lkocman — CHANGES_REQUESTED** [review]
I'd just replace 16.0 (which was hardcoded from before we had working release package) to zypper's $distver so zypper --releasever 16.1 can be used for future migrations. Otherwise looks good

**Comment by stacheldrahtje** [comment]
are now effected... I wondered why it was hardcoded indeed.

**Pull request closed without merging**